### DISCUSSION TITLE
Nested route groups are not allowed

### DISCUSSION THREAD

**Discussion by farshadfahimi** [adonisjs/core#1470] [Help] [answered]
I have some routes like below

```javascript
Route.group(() => {

    Route.group(() => {
        //some routes
    })

    // some routes
}).prefix('api')

```

write now I get an error that said `Nested route groups are not allowed`. 
did you think about to extend this feature for ability to create the nested group routs?

**Comment by RomainLanz** [comment, marked as answer]
Hey @farshadfahimi! 👋 

We don't plan to add this feature back in V4, it's already implemented in V5.
